src/message-dispatch.js

```diff
@@ -140,6 +140,14 @@ export default class MessageDispatch {
           }
         }
         break;
+      case "audiomode":
```

Comment:
Thoughts on adding this to the user preference screen?

Reply:
Originally was going to be, but held off for now because we don't actually know that this will help noise cancellation problems. Can easily re-enable it in the preferences if we prove that this helps.